doc/tutorials.md

```diff
@@ -239,40 +234,41 @@ sourmash gather -k 31 shakya-unaligned-contigs.sig genbank-k31.sbt.json
 
 This should yield:
 ```
-# running sourmash subcommand: gather
 loaded query: mqc500.QC.AMBIGUOUS.99.unalign... (k=31, DNA)
-loaded SBT genbank-k31.sbt.json
+loaded 0 signatures and 1 databases total.
```

Comment:
That's been fixed in #372, which was recently merged. It's because no signatures were loaded, only a database, but it always bugged me too (even though I wrote it).